pkg/object/apirule.go

```diff
@@ -64,19 +64,17 @@ func GetService(svcName string, port uint32) apigatewayv1beta1.Service {
 
 // WithService sets the Service of an APIRule.
 func WithService(host, svcName string, port uint32) Option {
-	return func(o metav1.Object) {
-		d := o.(*apigatewayv1beta1.APIRule)
+	return func(o *apigatewayv1beta1.APIRule) {
```

Comment:
```suggestion
	return func(r *apigatewayv1beta1.APIRule) {
```
nit: why o for APIRule?

Reply:
Done